samples/Java/README.md

```diff
@@ -11,14 +11,16 @@ Azure Event Hubs is a highly scalable data streaming platform and event ingestio
 
 ### Sending events
 
-* **Send** - The [Send](./Basic/Send) sample illustrates how to ingest events into your event hub.
+* **Send** - The [Send](./Basic/Send) sample illustrates the various options available with Event Hubs to ingest events.
 * **SendBatch** - The [SendBatch](./Basic/SendBatch) sample illustrates how to ingest batches of events into your event hub.
+* **SimpleSend** - The [SimpleSend](./Basic/SimpleSend) sample illustrates how to ingest events into your event hub.
```

Comment:
consider renaming this to 
**send** - this feels like what is shown in simple send.
**sendBatch**
**advancedSend** - add partition scenarios here - and provide guidance that - these types of sends comes with more partition control and lower availability.